### ISSUE TITLE
[Other - Accessibility] `role` not appropriate for this component. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#61157]
#### VFS team name
Authenticated Experience

#### VFS product name
My VA

#### VFS feature name
Notification design system component

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
As currently coded, each notification has a `role="alert"`. That role maps to an `assertive` ARIA live region and prompts screen readers to announce the notification as soon as the page loads. You don't get a sense for the context of the page you're on or even the number of notifications there are. Instead, you just get a string of notifications all announced in a row, without any easy way to distinguish them.

I feel like this isn't desirable behavior, especially when multiple notifications are on the page. It would be appropriate for an error alert or something where immediate action is required, but that's not the case here when users can choose to act on the notification, ignore it, dismiss it, etc. 

### Link, screenshot or steps to recreate
Steps to reproduce:
1. Activate a screen reader (easiest to test with NVDA on Windows, VoiceOver on Mac).
2. Log in to staging using test user 226.
3. After redirecting back to My VA, note that notifications are announced immediately on page load.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Remove `role="alert"` from the notification component. I think notifications as implemented on My VA should be sufficiently discoverable since they're grouped beneath the first H2 on the page. A screen reader user exploring the page should find them pretty quickly without needing anything more assertive.

What might be more useful for future work would be some kind of `aria-live` region that announces summary information after the page loads, eg. "You have 4 new notifications." That would indicate to the user that there's something for them to find, but doesn't push it too hard so as to interrupt what they're doing.

More broadly, there may need to be more conversation around the design system guidance for implementing notifications elsewhere on VA.gov. When and how a component is announced (or a summary "You have 4 new notifications" etc) could be different depending on the context and goals of a user in that context, or we could establish a model with My VA and say that's the canonically right way to do it.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by ajakabcin** [comment]
We'll address this in #61388 and #61399

**Comment by shiragoodman** [comment]
@ajakabcin @nickjg231 sorry, I am opening this back up. Please do not close out any Collab Cycle Staging Review issue tickets until the issue is resolved. Thank you.

**Comment by ajakabcin** [comment]
hey @shiragoodman I believe this is resolved by @nickjg231's [update](https://github.com/department-of-veterans-affairs/component-library/pull/741) posted above?

**Comment by ajakabcin** [comment]
sure thing @shiragoodman I think @nickjg231 assumed his [update](https://github.com/department-of-veterans-affairs/component-library/pull/741) above covered the entire issue - agreed this will be completely resolved when my team finishes #61399

**Comment by ajakabcin** [comment]
@shiragoodman I just followed up with @allisonlu and Nick's change actually fully resolves this so we can close it - guess there was some miscommunication as to whether my team or DST was going to handle this issue. We still have a second part of our ticket to do that resolves #61158, but this is done now

**Comment by shiragoodman** [comment]
thanks for the follow up @ajakabcin ! sounds good to me.
